src/rest/mormot.rest.mvc.pas

```diff
@@ -603,6 +603,9 @@   TMvcRunWithViews = class(TMvcRun)
   /// run TMvcApplication directly within a TRestServer method-based service
   // - this is the easiest way to host and publish a MVC Application, optionally
   // in conjunction with REST/AJAX client access
+

```

Comment:
please remove this comment generated by the IDE which breaks the main documentation comment

Reply:
Done.